### PR TITLE
Allows flexible configuration of database via ENV vars 

### PR DESCRIPTION
**Pull request by garytaylor**
Allows flexible configuration of database via ENV vars (for development and test environments) for docker configs
If ENV vars are not set, works as before

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -243,7 +243,7 @@
     t.integer  "income"
     t.boolean  "refund",                        null: false
     t.date     "date_fee_paid"
-    t.boolean  "probate"
+    t.boolean  "probate",                       null: false
```
Comment (leoapost) [inline db/schema.rb]: is this going to cause any issues?

Comment (garytaylor) [reply db/schema.rb]: Hmm, I never noticed that - I never changed it - my guess is that the schema.rb in master is wrong as I built this db from scratch.  Ill double check what migrations say

Comment (garytaylor) [reply db/schema.rb]: According to the migrations, this is correct - see db/migrate/20160308155249_add_online_application_tables.rb

Comment (leoapost) [reply db/schema.rb]: yes, that looks to be the case. Probably worth moving it to a separated commit?

**Pull request merged**